exec/java-exec/src/test/java/org/apache/drill/TestExampleQueries.java

```diff
@@ -33,6 +33,30 @@
 public class TestExampleQueries extends BaseTestQuery {
 //  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(TestExampleQueries.class);
 
+  @Test // DRILL-2915
```

Comment:
If this is fixing join ordering, shouldn't this be doing plan verification in addition to checking the results?